docs/docs.trychroma.com/pages/reference/cli.md

```diff
@@ -1,3 +1,17 @@
 ---
 title: CLI
 ---
+
+## Vacuuming
+
+Vacuuming shrinks and optimizes your database. **It does not need to be run regularly**. It blocks all reads and writes to your database while it's running, so it should only be run when absolutely necessary. In most cases, vacuuming will save very little disk space. We recommend shutting down your Chroma server before vacuuming (although it's not strictly required).
```

Comment:
In general it's the case that this *only* needs to be run once right? Since after it's run, we will automatically prune the WAL?

Reply:
yes
additionally, for people with consistent write load, vacuuming *may* improve query performance
but I doubt the effect will be very noticeable unless your collections are quite large

Reply:
We should communicate that in general, you should only need to do this once, we will tell you if you think you need to do it, and we do not anticipate that except in exceptional cases you will ever need to do it in future.

Reply:
I think "it does not need to be run regularly" covers most of this, but I can make it more explicit

Reply:
e07d01ec